src/remark/variablesPlugin.js

```diff
@@ -1,51 +1,64 @@
-const path = require("path");
-const visit = require("unist-util-visit");
+const path = require("path")
```

Comment:
I would suggest configuring your editor to allow semicolons for each line. By the way, I'm using `Prettier` - you can install it and then we'll be able to sync up our settings. Or, moreover, we can add `.prettierrc` and enable the same code styles for all of us.

Reply:
I have added prettier and configured WebStorm to use it for all formatting on save. It's configured to do so for all javascript and markdown oriented files. If you want to add that .prettierrc file, go for it. Personally, I would prefer we get rid of the semicolons, use single quotes when we can, and use tabs. But I don't care that much. It's nice to just be all consistent with each other. I really don't care for the way it formatted the 'visit' function call though - ugly!